lib/enkf/site_config.cpp

```diff
@@ -86,6 +86,12 @@
    xxx_site counterpart are stored.
  */
 
+
+#define RUN_MODE_PRE_SIMULATION_NAME  "PRE_SIMULATION"
```

Comment:
Would it be possible to move both these and `hook_manager.cpp`'s `defines` to a shared header? I would expect to find these definitions in `config_keys.hpp`.

Reply:
Agreed :+1: